source/1_engine/rex_engine/src/platform/win/diagnostics/win_debug.cpp

```diff
@@ -25,6 +25,12 @@ void rex::exit_confirm(int32 exitCode)
   }
 }
 
+void rex::output_debug_string(rsl::string_view str)
+{
+  OutputDebugStringA(str.data());
+  OutputDebugStringA("\n");
+}
```

Comment:
There is a Sink within spdlog that uses "OutputDebugString" could you convert this to using this logger instead.
You can register a separate logger just for this use case.

Reply:
No I can't, this code is meant to be able to be called before the logger is initialized